### PR TITLE
Feat/controllermaps

### PR DESCRIPTION
**Pull request by memori034**

### CODE REVIEW

**Comment by realjoshuau** [comment]
@memori034 Did someone review this? 
If someone did, make sure they mark it as approved on Github.

We require at least 1-2 reviews on non-minimal changes/PRs, and this seems like a sizeable enough change to warrant a review on first glance.

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -82,27 +83,31 @@ public class RobotContainer {
   private boolean isRed;
 
   private final CommandSwerveDrivetrain drivetrain = TunerConstants.DriveTrain;
-  private double MaxSpeed =
+
+  public double MaxSpeed =
       TunerConstants.kSpeedAt12VoltsMps; // kSpeedAt12VoltsMps desired top speed
-  private double MaxAngularRate = 1.5 * Math.PI; // My drivetrain
+  public double MaxAngularRate = 1.5 * Math.PI; // My drivetrain
```
Comment (chaoticthegreat) [inline src/main/java/frc/robot/RobotContainer.java:89]: keep private same for the rest

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -404,47 +425,64 @@ public void configureSwerve() {
...
-                        // negative Y (forward)
-                        .withVelocityY(
-                            -driver.getLeftX() * MaxSpeed) // Drive left with negative X (left)
-                        .withRotationalRate(
-                            driver.getRightTriggerAxis()
-                                * MaxAngularRate) // Drive counterclockwise with negative X
-                // (left)
-                ));
+                    azi.withTargetDirection(
+                        Rotation2d.fromDegrees(
```
Comment (chaoticthegreat) [inline src/main/java/frc/robot/RobotContainer.java:437]: use rotation2d math instead (new Rotation2d(x, y))

Comment (chaoticthegreat) [reply src/main/java/frc/robot/RobotContainer.java:437]: also I think if you dont specify velocity X and Y its not going to allow you to move while pressing right trigger

File: src/main/java/frc/robot/subsystems/swerve/AzimuthConstants.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+//
+// Use of this source code is governed by a 
+// license that can be found in the LICENSE file at
+// the root directory of this project.
+
+package frc.robot.subsystems.swerve;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public final class AzimuthConstants {
```
Comment (chaoticthegreat) [inline src/main/java/frc/robot/subsystems/swerve/AzimuthConstants.java:12]: I would just stick these inside swerve constants, but it's your choice.

**Review by chaoticthegreat — COMMENTED** [review]
nice

**Comment by realjoshuau** [comment]
iirc CTRE Swerve flips the orientation so idk if you need a red / blue angle, check in sim or irl

**Pull request merged**